src/syntax/select.spec.ts

```diff
@@ -1060,4 +1060,48 @@ describe('Select statements', () => {
             type: 'skip locked',
         }
     });
+
+    checkSelectLoc('select * from test as t', {
+        _location: { start: 0, end: 23 },
+        type: 'select',
+        from: [{
+            _location: { start: 14, end: 23 },
+            type: 'table',
+            name: {
+                _location: { start: 14, end: 23 },
+                name: 'test',
+                alias: 't',
+            }
+        }],
+        columns: [{
+            _location: { start: 7, end: 8 },
+            expr: {
+                _location: { start: 7, end: 8 },
+                type: 'ref',
+                name: '*',
+            }
+        }]
+    });
+
+    checkSelectLoc('select * from test t', {
+        _location: { start: 0, end: 20 },
+        type: 'select',
+        from: [{
+            _location: { start: 14, end: 20 },
+            type: 'table',
+            name: {
+                _location: { start: 14, end: 20 },
+                name: 'test',
+                alias: 't',
+            },
+        }],
+        columns: [{
+            _location: { start: 7, end: 8 },
+            expr: {
+                _location: { start: 7, end: 8 },
+                type: 'ref',
+                name: '*',
+            },
+        }]
+    });
```

Comment:
This test fails on `master`. The `end` positions there are 18 instead of 20, because the ` t`  part is omitted.